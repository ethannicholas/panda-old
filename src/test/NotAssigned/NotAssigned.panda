var a:Int
if false
    a := 5
Console.writeLine(a)