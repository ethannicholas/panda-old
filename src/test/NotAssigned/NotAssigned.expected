error: NotAssigned.panda:4:19: variable 'a' may not have been assigned
1 error
